aws/README.md

```diff
@@ -4,52 +4,94 @@ This directory contains the configuration files for the production infrastructur
 
 ## One-time setup
 
-Download the aws and ecs-cli tools from Amazon.
+Install `eksctl` and `kubectl` to interact with Amazon EKS and the Kubernetes control plane.
 
-Configure the aws CLI, specify `us-east-1` as the default region and enter your _Access keys_ generated from the [web console](https://console.aws.amazon.com/iam/home?region=us-east-1#/security_credentials).
+## Kubernetes setup
 
-```shell
-aws configure
-```
+Our cluster can be seen on the [AWS console](https://console.aws.amazon.com/eks/home?region=us-east-1#/clusters) but that console is pretty much useless as it shows no information on the cluster nodes.
+
+You can also list them with `eksctl get cluster`.
 
-Configure ecs-cli to work on our cluster:
+The cluster was originally created with the command:
 
 ```shell
-ecs-cli configure --cluster epid-dev --default-launch-type FARGATE --region us-east-1
+eksctl create cluster --name=epid --region=us-east-1 --fargate --version=1.16
 ```
 
-Create the cluster:
+The basic deployment/pods/services configuration looks like:
 
-```shell
-ecs-cli up
+```
+kubectl get deployments
```

Comment:
nit: could you include instructions on configuring kubectl to connect to our cluster? This is automatically done by kubectl (and probably eksctl) if you create a cluster, but not otherwise.

https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#accessing-for-the-first-time-with-kubectl

Reply:
Ah right sorry, yeah this was done for me when I created the cluster, I added a section in first-time set up to explain how to do this.
Please let me know if that worked for you.